addons/web/static/src/search/search_bar/search_bar.js

```diff
@@ -682,7 +686,7 @@ export class SearchBar extends Component {
     }
 
     onInputDropdownChanged(isOpen) {
-        if (!isOpen) {
+        if (!isOpen && !this.isDestroyed) {
```

Comment:
```suggestion
        if (!isOpen && status(this) === "mounted") {
```

Reply:
Could you add a test? There's another scenario to reproduce the issue, which does not involve doing browser back (which is kind of tricky in unit tests):
- mount a webclient
- execute an action with a list and a form view
- mock onRpc to slow down/block web_read
- click on a record
- while the web_read is blocked, click/type something in the search bar
- release web_read
- => crash because the callback is called when the component in unmounted